### DISCUSSION TITLE
ActivityWatch

### DISCUSSION THREAD

**Discussion by 42willow** [catppuccin/catppuccin#2405] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey :wave:,

I've closed this discussion since the original post doesn't include any links on how to theme the application or link to any source code of other themes being applied to the application.

Would be more than happy to re-open once the post is updated with the relevant links or instructions on how to theme it. 

Thanks!

**Reply by 42willow** [reply]
Sorry about that, I don't use ActivityWatch anymore and have completely forgotten about my train of thought when opening this. Good idea to close this discussion unless someone else wants to pick this up :)
